### ISSUE TITLE
[Bug]: awb_bad_kreuznach not working

### ISSUE THREAD

**Issue by Wishmaster77** [mampfes/hacs_waste_collection_schedule#1643]
### I Have A Problem With:

A specific source

### What's Your Problem

New to HA and I am trying to implement the awb_bad_kreuznach_de source but i get no entries in the calendar.
When using a different source (e.g. awb_mainz_bingen_de) everything works instantly so i would think that the base config in HA is correct.

The source was updated some month ago, maybe something has changed again. I know that the provider was quite late with updating the calendar due to some problems they had if i remember correctly.

(https://github.com/mampfes/hacs_waste_collection_schedule/pull/1036/commits/283af0c0273724ef87612f69f1c9e1c44593d59a)

### Source (if relevant)

awb_bad_kreuznach_de

### Logs

```Shell
no relevant logs
```

### Relevant Configuration

```YAML
waste_collection_schedule:
  sources:
    - name: awb_bad_kreuznach_de
      args:
        ort: "Roxheim"
        
sensor:
  - platform: waste_collection_schedule
    name: Papiertonne
    types:
      - Altpapier
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [ ] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by 5ila5** [comment]
fixed by #1648 
they changed the way they ship the dates the old way did not work anymore now I have to calculate the next collection and adjust on holidays. I hope everything works fine

This will be part of release 1.45.0 or is available now as master (git) version (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)

**Comment by Wishmaster77** [comment]
#1648 Can confirm this works now as expected.

Many Thanks for the quick fix
